### PR TITLE
Help/Blue Sky Page Updates

### PR DESCRIPTION
**Pull request by wpscholar**
## Proposed changes

- Make video image clickable and load the YouTube video when clicked (on both Help & Blue Sky)
- Update external links to open in a new tab
- Ensure that phone number links show the phone number
- Update tests

## Type of Change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: src/app/pages/blue-sky/style.scss [hunk 1]
```diff
@@ -340,6 +345,15 @@ $mobile-padding: 30px;
         margin-bottom: 2rem;
         font-weight: 300;
         text-align: center;
+
+        a {
+          text-decoration: none;
+          color: inherit;
+
+          &:hover {
+            text-decoration: underline;
```
Comment (0aveRyan) [inline src/app/pages/blue-sky/style.scss:354]: Link buttons don't receive underlines [per Bluebird UI](https://bluebirdui.com/?path=/docs/components-buttons--link-button)

File: src/app/pages/help/index.js [hunk 2]
```diff
@@ -130,7 +148,13 @@ const Help = () => {
 				<div className="resources-search">
 					<Search />
 					<div className="button-container">
-						<Button href="https://www.bluehost.com/resources/" isLink isSecondary>
+						<Button
+							href="https://www.bluehost.com/resources/"
+							isLink
```
Comment (0aveRyan) [inline src/app/pages/help/index.js:153]: The doubling up of `isLink` and `isSecondary` gets resolved in #61, but may need a rebase after this linting.

**Review by 0aveRyan — APPROVED** [review]

**Review by 0aveRyan — CHANGES_REQUESTED** [review]
See note about `isLink` underline styles

File: src/app/pages/blue-sky/style.scss (hunk 1 above)
Comment (wpscholar) [reply src/app/pages/blue-sky/style.scss:360]: That class only applied to the footer links (not buttons) in this instance.

File: src/app/pages/help/index.js (hunk 2 above)
Comment (wpscholar) [reply src/app/pages/help/index.js]: Updated

**Review by 0aveRyan — APPROVED** [review]
I think we need to set some time with Adam re: underline styles in general. 

The call-to-action in the blue sky pricing table is effectively a margin-less Link Button, so in that last review I'd flagged the underline style that got added. But, after reviewing Bluehost.com legacy, new pricing tables plus Bluerock Marketplace tables and the bluebird UI table's reference... some are underline and some are just color changes 🤷‍♂️ It's small, lets let this ride, but I'd love to just iron out some general frontend guidelines for these kinda things that aren't quite covered in Bluebird.

**Pull request merged**